fastlane/Fastfile

```diff
@@ -0,0 +1,19 @@
+# Import base_config from git
+import_from_git(url: 'https://github.com/rakutentech/ios-buildconfig.git')
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Build All"
+  lane :ci do |options|
+    xcversion(version: "~> 10.2")
```

Comment:
If xcode11 gets set in the `.travis.yml` this will need changed too.

Reply:
Sure, will update the same.